src/panel_gwalker/_gwalker.py

```diff
@@ -216,3 +287,12 @@ async def export(
                     f'Exporting {scope} chart(s) timed out.'
                 )
         return self._exports.pop(event_id)
+
+    def create_export_button(self)->ExportButton:
```

Comment:
I added this because it takes a lot of code to create this functionality. And it might be something users want to add to their apps.

I would for sure.